### PR TITLE
Use Ctrl+C to discard the input in console

### LINKED ISSUE #2762: Use Ctrl+C to discard the input in console
**Product:** Tarantool
**Since:** 2.10.0-beta2-168-ga8b3b267c
**Audience/target:** dev
**Root document:** https://www.tarantool.io/en/doc/latest/reference/interactive_console/
**SME:** @ locker

The signal SIGINT discards the input in console mode.
When tarantool executes with -e flag or runs as a daemon, SIGINT
kills the process and tarantool complains about it in log.
Requested by @vr009 in https://github.com/tarantool/tarantool/commit/a8b3b267c36500ea7800da5e86f97020850e92af.

### PR DESCRIPTION
**Pull request by xuniq**
Fixes #2762

### CODE REVIEW

File: doc/reference/interactive_console.rst [hunk 1]
```diff
@@ -120,3 +125,22 @@ call was ``console.set_default_output('lua')``.
...
+------------------
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 1
+
+        *   - Keyboard shortcut
+            - Effect
+
+        *  - ``CTL+C``
```
Comment (Totktonada) [inline doc/reference/interactive_console.rst]: It usually abbreviated as Ctrl, not Ctl AFAIK.

AFAIR, we should use some template for keyboard keys.

File: doc/reference/interactive_console.rst [hunk 2]
```diff
@@ -120,3 +125,22 @@ call was ``console.set_default_output('lua')``.
...
+           - Discard current input with the ``SIGINT`` signal in the console mode and
+             jump to a new line with a default prompt.
```
Comment (Totktonada) [inline doc/reference/interactive_console.rst]: But it shutdowns tarantool if a command is executed currently (when we don't wait for a user input).

File: doc/reference/interactive_console.rst (hunk 1 above)
Comment (xuniq) [reply doc/reference/interactive_console.rst]: Fixed typo. Thanks for noticing!

File: doc/reference/interactive_console.rst [hunk 3]
```diff
@@ -120,3 +125,27 @@ call was ``console.set_default_output('lua')``.
...
+        - Discard current input with the ``SIGINT`` signal in the console mode and
+          jump to a new line with a default prompt.
+
+    *   - ``CTRL+D``
+        - Quit Tarantool interactive console.
+
+..  important::
+
+    Keep in mind that ``CTRL+C`` shortcut will shut Tarantool down if there is any currently running command
+    in the console.
```
Comment (vr009) [inline doc/reference/interactive_console.rst]: I would add that SIGINT shuts the instance down if it is running in a daemon mode.

**Review by vr009 — APPROVED** [review]

File: doc/reference/interactive_console.rst (hunk 3 above)
Comment (xuniq) [reply doc/reference/interactive_console.rst]: Added. Thanks for the review!

**Pull request merged**